Mathlib/Algebra/Category/MonCat/FilteredColimits.lean

```diff
@@ -167,12 +166,14 @@ noncomputable instance colimitMulOneClass : MulOneClass (M.{v, u} F) :=
     one_mul := fun x => by
       refine Quot.inductionOn x ?_
       intro x
+      change 1 * M.mk _ _ = M.mk _ _
```

Comment:
These `show` look suspicious. Though there is already some `show` in the code before so it is probably not your fault and I don't mind if you choose not to fix them to prevent the diff from getting too messy.